src/components/WheelPicker/__tests__/index.spec.tsx

```diff
@@ -43,10 +43,10 @@ describe('WheelPicker', () => {
       const renderTree = render(<TestCase/>);
       const driver = WheelPickerDriver({renderTree, testID});
 
-      driver.moveToItem(4, NUM_OF_ROWS, ITEM_HEIGHT);
```

Comment:
We can remove the ITEM_HEIGHT and test the defaults or maybe add a test case for the defaults

Reply:
I think adding a test case is the best way